scalafmt-tests/src/test/scala/org/scalafmt/ToHoconSuite.scala

```diff
@@ -1,10 +1,6 @@
 package org.scalafmt
 
-import metaconfig.Conf
-import metaconfig.ConfEncoder
-import org.scalafmt.config.Config
-import org.scalafmt.util.DiffAssertions
-import org.scalatest.funsuite.AnyFunSuite
+import munit.FunSuite
 
-class ConfDiffSuite extends AnyFunSuite with DiffAssertions {}
-class ToHoconSuite extends AnyFunSuite with DiffAssertions {}
+class ConfDiffSuite extends FunSuite {}
```

Comment:
WTF? Is this classes necessary?

Reply:
Is there any issue if we remove these?

Reply:
I think that no. I guess they was created as TODO-markers but were never finished

Reply:
I think those test suites got moved upstream into the metaconfig repo. We can remove them here.

Reply:
done